core/templates/pages/exploration-editor-page/statistics-tab/statistics-tab.component.ts

```diff
@@ -39,13 +39,14 @@ require(
 angular.module('oppia').component('statisticsTab', {
   template: require('./statistics-tab.component.html'),
   controller: [
-    '$q', '$scope', '$uibModal', 'AlertsService', 'ComputeGraphService',
+    '$q', '$rootScope', '$scope', '$uibModal',
+    'AlertsService', 'ComputeGraphService',
     'ExplorationDataService', 'ExplorationStatsService',
     'ReadOnlyExplorationBackendApiService', 'RouterService',
     'StateInteractionStatsService', 'StatesObjectFactory',
     'UrlInterpolationService',
     function(
-        $q, $scope, $uibModal, AlertsService, ComputeGraphService,
+        $q, $rootScope, $scope, $uibModal, AlertsService, ComputeGraphService,
```

Comment:
Add a new line after `$uibmodal` to match the format in lines 42-43.

Reply:
done